client/src/main/java/com/paypal/selion/configuration/Config.java

```diff
@@ -526,6 +541,12 @@ public static synchronized void setConfigProperty(ConfigProperty configProperty,
         SITE_LOCALE("siteLocale", "US", false),
 
         /**
+         * Site will show country used for tests.<br>
```

Comment:
My bad, I did not update JavaDoc here. It should be:

        /**
         * mobile platform test suite is designed for (IOS or ANDROID)<br>
         * Default is set to <b>UNDEFINED</b>
         */
        MOBILE_PLATFORM("mobilePlatform", "UNDEFINED", false),